### PR TITLE
i18n(es): Translated `frontmatter.md` 

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Translated `frontmater.md` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+Puedes agregar etiquetas adicionales a la etiqueta `<head>` de tu página usando el campo `head` del frontmatter. Esto significa que puedes agregar estilos personalizados, metadatos u otras etiquetas a una sola página. Similar a la [opción global `head`](/reference/configuration/#head).
+
+```md
+---
+title: Acerca de nosotros
+head:
+  # Usa una etiqueta <title> personalizada
+  - tag: title
+    content: Custom about title
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/frontmatter.md:49]: No estoy seguro pero creo que esto se podria traducir

**Review by Waxer59 — COMMENTED** [review]
Buena PR te dejo una sugerencia 🤠

File: docs/src/content/docs/es/reference/frontmatter.md (hunk 1 above)
Comment (delucis) [reply docs/src/content/docs/es/reference/frontmatter.md:49]: Agreed!

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+**tipo:** [`HeadConfig[]`](/reference/configuration/#headconfig)
+
+Puedes agregar etiquetas adicionales a la etiqueta `<head>` de tu página usando el campo `head` del frontmatter. Esto significa que puedes agregar estilos personalizados, metadatos u otras etiquetas a una sola página. Similar a la [opción global `head`](/reference/configuration/#head).
+
+```md
+---
+title: Acerca de nosotros
+head:
+  # Usa una etiqueta <title> personalizada
+  - tag: titúlo
```
Comment (delucis) [inline docs/src/content/docs/es/reference/frontmatter.md:48]: The `tag` is the HTML element, so this should still be title for the `<title>` element:

```suggestion
  - tag: title
```

**Comment by dreyfus92** [comment]
Thanks alot @delucis & @Waxer59 for your reviews 😁

**Review by Waxer59 — COMMENTED** [review]
LGTM ✔️

**Pull request merged**